src/styles/styles.css

```diff
@@ -0,0 +1,65 @@
+body {
```

Comment:
This looks like the same styles that are in board.css, this should just hold global styles. Sounds like it might be empty at this point unless there are some that you think can be shared with other components.